dashboard/app/views/admin_search/find_students.html.haml

```diff
@@ -43,15 +43,19 @@
   = paginate @users, total_pages: (@total_count / AdminSearchController::MAX_PAGE_SIZE.to_f).ceil
   %table.users
     %tr
-      - ['ID', 'Name', 'Email', 'Provider', 'Deleted At', 'Undelete'].each do |field|
+      - ['ID', 'Name', 'Email', 'Provider', 'Can Share?', 'Deleted At', 'Undelete'].each do |field|
         %th
           %span= field
     - @users.each do |user|
       %tr
-        - [:id, :name, :email, :provider, :deleted_at].each do |field|
+        - [:id, :name, :email, :provider].each do |field|
           %td
             %span= user[field]
+        %td
+          %span= !user.sharing_disabled?
         - if user.deleted?
+          %td
+            %span= user[:deleted_at]
```

Comment:
Showing the `deleted_at` column wasn't behind the `if user.deleted?` check before. Intentional?

Reply:
I moved it on purpose, but maybe I shouldn't have. Are you thinking about a scenario where the user is deleted, then undeleted and we still want to know the timestamp for when they were initially deleted?

Reply:
Just wanted to make sure it was on purpose and tested as such, since it wasn't called out anywhere.  Seems fine.